cmd/plural/init.go

```diff
@@ -21,10 +21,21 @@ import (
 	"github.com/pluralsh/plural/pkg/wkspace"
 )
 
-func handleInit(c *cli.Context) error {
+const DemoingErrorMsg = "You're currently running a gcp demo cluster. Spin that down at https://app.plural.sh/shell before beginning a local installation"
+
+func (p *Plural) handleInit(c *cli.Context) error {
+	p.InitPluralClient()
 	gitCreated := false
 	repo := ""
 
+	me, err := p.Me()
```

Comment:
I'm pretty sure this needs to be done after `handleLogin` (a fresh user won't have a way to query the api before)

Reply:
yes, you are right